pkgs/by-name/ya/yandex-music/ym_info.json

```diff
@@ -1,14 +1,14 @@
 {
   "ym": {
-    "version": "5.18.2",
-    "exe_name": "Yandex_Music_x64_5.18.2.exe",
-    "exe_link": "https://music-desktop-application.s3.yandex.net/stable/Yandex_Music_x64_5.18.2.exe",
-    "exe_sha256": "a513eb6aa4bd05bae8503f5578a8e91d778794ffe4eb3a450040903570106941"
+    "version": "5.23.2",
+    "exe_name": "Yandex_Music_x64_5.23.2.exe",
+    "exe_link": "https://music-desktop-application.s3.yandex.net/stable/Yandex_Music_x64_5.23.2.exe",
+    "exe_sha256": "3e15e650657dfbdcdf459b9fc37155be93c6a952c0079d65af3eca268be6f82f"
   },
   "electron": {
```

Comment:
This section doesn't seem to be used.

Reply:
Yes, this file downloaded from original repository as is by updater script. I've modified  script to drop this section.